### ISSUE TITLE
Shard of the Fallen Star trinket is not scaling at 100% of spell dmg

### ISSUE THREAD

**Issue by theloras** [elysium-project/server#700]
**BUGGED BEHAVIOR**
https://cdn.discordapp.com/attachments/253215035613118467/312861310733778947/shard_of_the_fallen_star.jpg

**CORRECT BEHAVIOR**
Shard of the Fallen Star
Binds when picked up
Trinket	
Requires Level 60
Use: Calls down a meteor, burning all enemies within the area for 400 to 443 total Fire damage. (cooldown 3 min)

It should scale at 100% of spell damage.

Xolkar 20/0/31 Paladin PvP - https://youtu.be/Pk_kHHI0t4o?t=83 - Uploaded on Aug 13, 2006 - Xolkar is using Shard of the Fallen Star. You see it crit for 1167 when he pops his ZHC trinket.

https://cdn.discordapp.com/attachments/253215035613118467/312863220664893441/shard_crit.jpg

**SOURCES**
http://db.vanillagaming.org/?item=21891#wh-comments

By Wirxaw on 09/22/2009 (Patch 3.1.3)
Pretty useless trinket without spell power scaling. If Blizz hadn't fix it, it could scale like: orig. WoW - ~500 spell power = 600-700 damage(nice for that time), TBC - ~1000 spell power = 1000-1200(also quite fine) and now in WotLK it could be like +2000 SP = ~2400 damage. Not that much but still could have done some harm. It would never be OP with even 100% scaling. Now it's just a waste of time to grind it for the sake of awesomeness._

By Archangel on 01/25/2007 (Patch 2.0.6)
This has about a .2% chance to drop off of AQ 40-man trash mobs. I hope to get level 70 and 5-man grind these guys for the trinket ^^ Just for the coolness factor.

Unless this was nerfed, it is 100% affected by spell damage. Best I can figure from what I've seen in videos, it runs roughly 70% coefficient. Note, however, the damage is dispersed amungst all targets. So, if it will deal 1000 damage to a single target, it will deal 500 each to 2 targets.

http://www.youtube.com/watch?v=cvWFOvGfbH8

Enjoy!

**Comment by fidbugs** [comment]
This is not the case, you have miss some things in both videos. 

In Xolkar one at 1:29 you can see that 1167 crit dammage is actualy Judgement of Command on stuned target, while Shard of fallen star hits for 321 even when under ZHC trinket. That is visible in combat log:

https://www.youtube.com/watch?v=Pk_kHHI0t4o&feature=youtu.be&t=89

https://gyazo.com/97ee27cfea76a25f99ece07b1bb24d04

Also in the video with Dernos(Spelldmg paladin in action - the one with better resolution where you can actually see combat log), you can see him at 2:44 hitting with Shard and ZHC trinket for just 475(keep in mind 10% more damage on warrior from berserker stance):

https://www.youtube.com/watch?v=ay7RU0N-44Y&feature=youtu.be&t=164

Also there is other occasions where trinket is used, with full T2 and ZHC(sometimes even Power Infusion and battleground damage increase buff - Berserking) that shows no scaling:

@ 1:07 333 hit(ZHC+PI) 

@ 1:17 487 hit(ZHC+PI)

@ 1:37 431 hit(ZHC)

And so on.

To sum up. This trinket should not scale with spell damage from gear, as is proven in mentioned videos, the comment from wowiki are false and misleading, we have pure vanilla evidence that of no scaling with spell damage.

**Correct behaviour**

Shard of falen Star shoud not scale with spell damage.

**Comment by theloras** [comment]
sometimes even I get stuff wrong
